srml/im-online/src/lib.rs

```diff
@@ -174,7 +174,7 @@ decl_event!(
 decl_storage! {
 	trait Store for Module<T: Trait> as ImOnline {
 		/// The block number when we should gossip.
-		GossipAt get(gossip_at) config(): T::BlockNumber;
+		GossipAt get(gossip_at) build(|_| Zero::zero()): T::BlockNumber;
```

Comment:
```suggestion
		GossipAt get(gossip_at) : T::BlockNumber;
```

Zero is the default value for numeric types.